examples/Cloudy/Cloudy_emulate_sample.jl

```diff
@@ -0,0 +1,291 @@
+        # - θ: in unconstrained space
+        ϕ_true = load(data_save_file)["truth_input_constrained"]
+        θ_true = transform_constrained_to_unconstrained(priors, ϕ_true)
+
+    else
+        error("File not found: $data_save_file. Please run 'Cloudy_calibrate.jl' first.")
+
+    end
+
+    param_names = get_name(priors)
+    n_params = length(ϕ_true) # input dimension
+    Γy = ekiobj.obs_noise_cov
+
+    cases = [
+        "rf-scalar",
+        "gp-gpjl"  # Veeeery slow predictions
+    ]
+
+    # Specify cases to run (e.g., case_mask = [2] only runs the second case)
+    case_mask = [1, 2]
+
+    # These settings are the same for all Gaussian Process cases
+    pred_type = YType() # we want to predict data
+
+    # These settings are the same for all Random Feature cases
+    n_features = 600
+    nugget = 1e-8
+    optimizer_options = Dict(
+        "verbose" => true,
+        "scheduler" => DataMisfitController(terminate_at = 100.0),
+        "cov_sample_multiplier" => 1.0,
+        "n_iteration" => 20,
+    )
+
+    # We use the same input-output-pairs and normalization factors for 
+    # Gaussian Process and Random Feature cases
+    input_output_pairs = get_training_points(ekiobj,
+                                             length(get_u(ekiobj))-1)           norm_factors = get_standardizing_factors(get_outputs(input_output_pairs))
+    for case in cases[case_mask]
+
+        println(" ")
+        println("*********************************\n")
+        @info "running case $case"
+
+        if case == "gp-gpjl"
+
+            @warn "gp-gpjl case is very slow at prediction"
+            gppackage = GPJL()
+            # Kernel is the sum of a squared exponential (SE), Matérn 5/2, and
+            # white noise
+            gp_kernel = SE(1.0, 1.0) + Mat52Ard(zeros(3), 0.0) + Noise(log(2.0))
+            gaussian_process = GaussianProcess(
+                gppackage;
+                kernel = gp_kernel,
+                prediction_type = pred_type,
+                noise_learn = false,
+            )
+
+            # The data processing normalizes input data, and decorrelates
+            # output data with information from Γy
+            emulator = Emulator(
+                gaussian_process,
+                input_output_pairs,
+                obs_noise_cov = Γy,
+                normalize_inputs = false,
+                standardize_outputs = true,
+                standardize_outputs_factors = vcat(norm_factors...),
+            )
+
+        elseif case == "rf-scalar"
+
+            kernel_structure = SeparableKernel(
+                LowRankFactor(n_params, nugget),
+                OneDimFactor()
+            )
+
+            srfi = ScalarRandomFeatureInterface(
+                n_features,
+                n_params,
+                kernel_structure = kernel_structure,
+                optimizer_options = optimizer_options,
+            )
+
+            retained_svd_frac = 1.0
+
+            emulator = Emulator(
```

Comment:
Agreed! I re-structured the example such that the `Emulator` construction is now the same for both RF and GP, with the `mlt` input holding the information on which tool is used. I had set the normalization of inputs to `false` for the GP emulator because for some reason this improves the emulator performance substantially. Do you have an explanation for this? If we want to maximize the quality of the emulator I will have to set normalize to `true` or `false` depending on whether we're dealing with RF or GP. This would be less elegant than having the `mlt` be the only difference between the two in terms of the `Emulator` construction...do you have thoughts on why for the GP case normalizing the inputs leads to worse results than not normalizing them?